src/Bicep.LangServer/Telemetry/TelemetryConstants.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Bicep.LanguageServer.Telemetry
+{
+    public class TelemetryConstants
+    {
+        public const string CommandName = "bicep.Telemetry";
+
+        public class EventNames
```

Comment:
[nit] static

Reply:
Updated as part of this commit:
https://github.com/Azure/bicep/commit/43337953f0cb9fe0d8c4386e00322b231ce9e766